tsctl

```diff
@@ -364,6 +365,49 @@ class CreateTimelineFromCsv(CreateTimelineBase):
         self.create_searchindex(timeline_name, index_name)
 
 
+class PurgeTimeline(Command):
+    """Delete timeline permanently from Timesketch and Elasticsearch."""
+    option_list = (
+        Option(u'--index', u'-i', dest=u'index_name', required=True),
+    )
+
+    def __init__(self):
+        super(PurgeTimeline, self).__init__()
+
+    def run(self, index_name):
+        """Delete timeline in both Timesketch snd Elasticsearch.
+
+        Args:
+            index_name: The name of the index in Elasticsearch
+        """
+        index_name = unicode(index_name.decode(encoding=u'utf-8'))
```

Comment:
It is needed because otherwise SQLAlchemy get's unhappy because the SQL schema enforces unicode only.